user_guide/intro_users_and_benefits.rst

```diff
@@ -0,0 +1,163 @@
+
+
+
+Get a 360-Degree view of your customers activity
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Centralizing information about your customers and their activities is at the heart of OroCRM. In order to learn how 
+customer accounts and contacts are managed in OroCRM, please refer to the 
+:ref:`Common Features <user-guide-common-features>` section.
+
+
+Organize information and make informed decisions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+OroCRM’s business intelligence tools will help to organize data and improve the decision making process:
+
+- Use predefined widgets to create informative dashboards to display metrics, pipelines, and gain quick access to 
+  regularly used activities
+
+- Analyze and visualize historical data and trends with reports
+
+- Narrow down the exact segment of data you’re looking for with segmentation 
+
+Please refer to the :ref:`Business Intelligence Tools section <business-intelligence>` of the User Guide of this 
+guide to learn more about how OroCRM can be used to analyze information.
+
+
+Communicate and Delegate
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Any CRM user can benefit from OroCRM’s productivity tools which allow any authorized user to:
+
+- Create calendar events for yourself or other users
+
+- Create templates with merge fields or write custom emails in the fly
+
+- Log calls
+
+- Assign tasks to yourself or to others 
+
+- Make notes and comments on different entities within OroCRM
+
+See the :ref:`Productivity Tools section <user-guide-productivity-tools>` of the User Guide to learn more about how OroCRM can help 
+to improve communication and task management within the company.
+
+
+Tools for Sales Representatives
+-------------------------------
+
+Sales representatives have many responsibilities which range from lead generation to closing a sale with a prospective 
+client.  Throughout the process notes are taken and needs are captured.  OroCRM helps sales representatives by providing 
+methods to store this information and by adding structure where needed to simplifying their work with the following 
+capabilities:
+
+
+- 360 degree view of a customer that includes past communication and notes
+
+- Ability to get in touch with customers directly from OroCRM with Google Hangouts or email
+
+- Fully customizable workflows and processes to provide structure to the sales process 
+
+- Tools to collect and process requests received via email or from a website form
+
+Please see the :ref:`Sales Tools section <user-guide-sales-index>` of the User Guide for more details.
+
+  
+Tools for Marketing Associates 
+------------------------------
+
+Marketing associates are responsible for growing a business's customer base.  This is sometimes done through email 
+marketing, customer segmentation, and campaign analysis.  OroCRM facilitates all of these actions and more by providing:
+
+- Segmentation on a specific set of customers by nearly any field  in OroCRM
```

Comment:
Edit: - Segmentation of a specific set of customers according to nearly any field in OroCRM